app/code/core/Mage/CatalogSearch/Model/Layer.php

```diff
@@ -99,6 +99,17 @@ protected function _prepareAttributeCollection($collection)
         return $collection;
     }
 
+    /**
+     * Filter which attributes are included in getFilterableAttributes
+     *
+     * @param Mage_Catalog_Model_Resource_Eav_Attribute $attribute
+     * @return  bool
+     */
+    protected function _filterFilterableAttributes($attribute)
```

Comment:
```suggestion
     */
    protected function _filterFilterableAttributes(Mage_Catalog_Model_Resource_Eav_Attribute  $attribute): bool
```

Reply:
Are we doing typed returns now, it would break 7.4 compatibility or has that been dropped?

Reply:
it would break 7.4 only for mixed types, which we can't use yet, but if it's a single type it should work fine:

https://onlinephp.io/c/f2834